### PR TITLE
Feat/prettier imports

### PR DESCRIPTION
**Pull request by IsaacGemal**

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -1,7 +1,14 @@
 {
-  "tabWidth": 2,
-  "useTabs": false,
```
Comment (huntcsg) [inline .prettierrc:3]: Do we want to retain this?

File: package.json [hunk 2]
```diff
@@ -7,7 +7,8 @@
     "dev": "concurrently \"tsc -b --watch --preserveWatchOutput\" \"vite\"",
     "build": "tsc -b && vite build",
     "lint": "eslint .",
-    "preview": "vite preview"
+    "preview": "vite preview",
+    "format": "prettier --write \"src/**/*.{ts,tsx}\""
```
Comment (huntcsg) [inline package.json:11]: love it 🚀

File: .prettierrc (hunk 1 above)
Comment (IsaacGemal) [reply .prettierrc:3]: I'll get rid of that I think

Comment (IsaacGemal) [reply .prettierrc:3]: We weren't using prettier before to lint, so I don't think it matters. For now this is just to fix the imports

Comment (IsaacGemal) [reply .prettierrc:3]: We can always revisit it later.

**Review by huntcsg — APPROVED** [review]

**Pull request merged**